packages/renderer/src/stores/kubernetes-contexts-state.ts

```diff
@@ -246,3 +247,15 @@ export const kubernetesCurrentContextRoutesFiltered = derived(
   [routeSearchPattern, kubernetesCurrentContextRoutes],
   ([$searchPattern, $routes]) => $routes.filter(route => findMatchInLeaves(route, $searchPattern.toLowerCase())),
 );
+
+// Port Forwarding
+
+export const kubernetesCurrentContextPortForwards = readable<UserForwardConfig[]>([], set => {
+  window.getKubernetesPortForwards().then(value => {
+    set(value);
+  });
+  window.events?.receive('kubernetes-port-forwards-update', (value: unknown) => {
+    set((value ?? []) as UserForwardConfig[]);
```

Comment:
question: based on where the event is fired it looks like it is always a value and we don't need the `?? []` ? We don't do it in the only other place we have this pattern in kubernetesContextsState.

Reply:
Yes, you are right, it was the case at some point, but was fixed, removed!